overlord/devicestate/devicestate_install_api_test.go

```diff
@@ -42,6 +42,7 @@ import (
 	"github.com/snapcore/snapd/gadget/install"
 	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/overlord/devicestate"
+	installLogic "github.com/snapcore/snapd/overlord/install"
```

Comment:
Maybe overlordInstall instead?

Reply:
Fwiw, I like this suggestion

Reply:
I haven't seen that style of aliasing, longer term I hope basically only overlord/install will import gadget/import so whatever we do we shouldn't need to live with it forever